app/assets/stylesheets/application.scss

```diff
@@ -1,5 +1,7 @@
 @import 'bootstrap';
+@import 'deja-vu-font';
 @import 'sul-variables';
+@import 'mixins';
```

Comment:
@drh-stanford The mixin for `h3-index-title-font` was just added with `searchworks-mixins.scss` in #74. Would you mind pointing this import to `searchworks-mixins` instead? An import order like this should do the trick:
```
@import 'deja-vu-font';
@import 'sul-variables';
@import 'bootstrap-variables';
@import 'bootstrap';
@import 'searchworks-mixins';
```

Once that's done, you can delete `mixins.scss`.